packages/ra-core/src/i18n/Translate.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { TestTranslationProvider } from './TestTranslationProvider';
+import { Translate } from './Translate';
+
+export default {
+    title: 'ra-core/i18n/Translate',
+};
+
+export const Basic = () => (
+    <TestTranslationProvider
+        messages={{
+            custom: {
+                myKey: 'My Translated Key',
+            },
+        }}
+    >
+        <Translate i18nKey="custom.myKey" />
+    </TestTranslationProvider>
+);
+
+export const NoTranslation = () => (
+    <TestTranslationProvider messages={{}}>
+        <Translate i18nKey="custom.myKey" />
+    </TestTranslationProvider>
+);
+
+export const NoTranslationWithChildren = ({
+    isChildrenANode = false,
```

Comment:
I'd make 2 stories instead of one - I think the argument is a bit artificial here.